### DISCUSSION TITLE
Multiple editors example

### DISCUSSION THREAD

**Discussion by jloganolson** [ueberdosis/tiptap#1408] [General Questions] [answered]
Hey there, I stumbled upon this part of the repo:
https://github.com/ueberdosis/tiptap/blob/7bf4c1d11ce4c36ad2846c4a15491ef8b649280d/docs/src/docPages/experiments/multiple-editors.md

Given there is no mention of this example in the docs (on .dev or main readme), is this a failed/wip example or the "correct"/"recommended" way of doing something like this?

**Comment by jloganolson** [comment, marked as answer]
https://www.tiptap.dev/experiments/multiple-editors/#multiple-editors <-d'oh sorry found it in docs

**Reply by hanspagel** [reply]
Yes! It shows how to store the content of multiple editors in a single Y.js document. Is that what you’re looking for?

**Reply by jloganolson** [reply]
Yes, though now i'm realizing my form has a variable number of nested fields (e.g. x number of contacts with x number of phone #s if this was an address book) - to make that work using Yjs/collaboration, my first guess is to keep a YMap of the current fields (so we can create/remove them across users). Does that make sense? If so, is it possible to "tack on" a YMap to the ydoc I'm using with these tiptap editors?

**Reply by hanspagel** [reply]
That should be possible! You can nest Y.js data types. Not sure how exactly though, would need to fiddle around with it.
